generics/middleware/authenticator.js

```diff
@@ -115,6 +115,13 @@ module.exports = async function (req, res, next) {
     return res.status(401).send(respUtil(rspObj));
   }
 
+   if ((req.path.includes("createGesture")) || (req.path.includes("createEmoji")) && (req.headers["internal-access-token"] !== process.env.INTERNAL_ACCESS_TOKEN)) {
```

Comment:
@deepakr123 - Refer to sunbird service implementation as - https://github.com/shikshalokam/sl-sunbird-service/blob/master/generics/middleware/authenticator.js